eip-template.md

```diff
@@ -15,7 +15,7 @@ This is the suggested template for new EIPs.
 
 Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
 
-The title should be 44 characters or less.
+The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category. 
```

Comment:
Should also be added to EIP-1, otherwise we get into this situation of EIP-1 and the template disagreeing again (like with simple summary).

Reply:
Added text to EIP-1.